doc/intl.md

```diff
@@ -0,0 +1,192 @@
+
+This document describes the current state of Android implementation of Intl APIs(formally ECMA-402 specification) in Hermes JavaScript engine. ECMA-402 is still evolving and the latest iteration is [7th edition](https://402.ecma-international.org/7.0/) which was published in June 2020. Each new edition is built on top of the last one and typically adds new capabilities typically as,
+- New Intl service constructors (for e.g. Intl.Collator, Intl.NumberFormat etc...) or extending existing ones by accepting more parameters
+- New functions or properties in the Intl object prototypes (for e.g. Intl.Collator.prototype.compare)
+- New locale aware functions in standard Javascript object prototypes (for e.g. String.prototype.localeCompare)
+
+One standard internationalization implementation strategy followed by other engines, is to bundle a library and data (typically [icu](http://site.icu-project.org/)) along with the application package. This guarantees deterministic functionality at the cost of applications package bloat.  We decided to consume the Android platform libraries and data for space efficiency, but at the cost of some variance in behaviour across  Android platforms.
+
+# ECMA-402 Compliance
+
+## Supported
+
+- Intl.Collator
+  - Intl.Collator.supportedLocalesOf
+  - Intl.Collator.prototype.compare
+  - Intl.Collator.prototype.resolvedOptions
+
+- Intl.NumberFormat
+  - Intl.NumberFormat.supportedLocalesOf
+  - Intl.NumberFormat.prototype.format
+  - Intl.NumberFormat.prototype.formatToParts
+  - Intl.NumberFormat.prototype.resolvedOptions
+
+- Intl.DateTimeFormat
+  - Intl.DateTimeFormat.supportedLocalesOf
+  - Intl.DateTimeFormat.prototype.format
+  - Intl.DateTimeFormat.prototype.formatToParts
+  - Intl.DateTimeFormat.prototype.resolvedOptions
+
+- Intl.getCanonicalLocales
+
+- String.prototype
+  - localeCompare
+  - toLocaleLowerCase
+  - toLocaleUpperCase
+
+- Array.prototype
+  - toLocaleString
+
+- Number.prototype
+  - toLocaleString
+
+- Date.prototype
+  - toLocaleString
+  - toLocaleDateString
+  - toLocaleTimeString
+
+## Not yet supported
+
+- [Intl.PluralRules](https://tc39.es/ecma402/#pluralrules-objects)
+
+- [Intl.RelativeTimeFormat](https://tc39.es/ecma402/#relativetimeformat-objects)
+
+- [Intl.DisplayNames](https://tc39.es/proposal-intl-displaynames/#sec-intl-displaynames-constructor)
+
+- [Intl.ListFormat](https://tc39.es/proposal-intl-list-format/#sec-intl-listformat-constructor)
+
+- [Intl.Locale](https://tc39.es/ecma402/#sec-intl-locale-constructor)
+
+- Intl.DateTimeFormat properties
+   - [dateStyle/timeStyle](https://tc39.es/proposal-intl-datetime-style/)
+   - [dayPeriod](https://github.com/tc39/ecma402/issues/29)
+   - [fractionalSecondDigits](https://github.com/tc39/ecma402/pull/347)
+- [BigInt.prototype.toLocaleString](https://tc39.es/ecma402/#sup-bigint.prototype.tolocalestring)
+
+## Excluded
+
+- [formatMatcher](https://tc39.es/ecma402/#sec-basicformatmatcher) property in Intl.DateTimeFormat.

```

Comment:
Could you explain with more details like what you did in `*.docx`? You also mentioned that Firefox/Chrome does not seem to respect this.